safebrowser.go

```diff
@@ -391,6 +393,18 @@ func (sb *SafeBrowser) WaitUntilReady(ctx context.Context) error {
 // If an error occurs, the caller should treat the threats list returned as a
 // best-effort response to the query. The results may be stale or be partial.
 func (sb *SafeBrowser) LookupURLs(urls []string) (threats [][]URLThreat, err error) {
+	threats, err = sb.LookupURLsContext(context.Background(), urls)
+	return threats, err
+}
+
+// LookupURLsContext looks up the provided URLs, with the ability to cancel the
+// request via a provided context. It is safe to call this method concurrently.
```

Comment:
Comment that LookupURLsContext is canceled either when ctx itself is canceled or after a timeout of Config.RequestTimeout.

The policy chosen is reasonable to me, but I can imagine other policies too (e.g., Config.RequestTimeout being completely ignored for this method).

Reply:
Done